infra/gcp/infra.yaml

```diff
@@ -333,6 +333,7 @@ infra:
       k8s-staging-cri-tools:
       k8s-staging-csi:
       k8s-staging-csi-secrets-store:
+      k8s-staging-csi-vsphere:
```

Comment:
Can we undo the change to this file? The images will be pushed to a different location and I'll open a PR for it.

Reply:
@upodroid reverted change in `infra/gcp/infra.yaml`